packages/medusa/src/services/order.ts

```diff
@@ -1412,6 +1412,27 @@ class OrderService extends TransactionBaseService {
     order: Order,
     totalsFields: string[] = []
   ): Promise<Order> {
+    if (totalsFields.some((field) => ["subtotal", "total"].includes(field))) {
+      const calculationContext =
+        await this.totalsService_.getCalculationContext(order, {
+          exclude_shipping: true,
+        })
+      order.items = await Promise.all(
+        (order.items || []).map(async (item) => {
+          const itemTotals = await this.totalsService_.getLineItemTotals(
+            item,
+            order,
+            {
+              include_tax: order.region?.automatic_taxes,
```

Comment:
thought: when we are calculating LineItemTotals for an order we will always have taxes so would suggest
```suggestion
              include_tax: true,
```

Reply:
You are right 👌 i ll change it